src/Uno.UI/UI/Xaml/Internal/InputManager.Pointers.cs

```diff
@@ -71,7 +71,7 @@ public PointerManager(InputManager inputManager)
 				UIElement.PointerReleasedEvent,
 				new PointerEventHandler((snd, args) => ProcessPointerUp(args, false)),
 				handledEventsToo: true);
-#if __WASM__
+#if __WASM__ && !UNO_HAS_MANAGED_POINTERS
```

Comment:
Always false?

Reply:
Indeed, fix the bug related to this and completely removed conditional code.